deploy/examples/cluster-on-pvc-minikube.yaml

```diff
@@ -7,6 +7,11 @@
 # For example, to create the cluster:
 #   kubectl create -f crds.yaml -f common.yaml -f operator.yaml
 #   kubectl create -f cluster-on-pvc-minikube.yaml
+#
+# or by using the using the tests/scripts/create-dev-cluster.sh and the following minikube configuration:
+
+#  --disk-size=20g --extra-disks=4
```

Comment:
--extra-disk=3?

Reply:
@parth-gr 4 disks are used: 1 for the mon storage and the remaining 3 are for OSDs